scripts/add-validator/add-validator.sh

```diff
@@ -36,12 +36,12 @@ done
 echo $domain
 echo $id
 
-sed -i "" "s|PROVIDER=.*|PROVIDER=wss://$domain:9945|" scripts/validator/.env
```

Comment:
Revert all changes not related to path updates in this sh

Reply:
updated